_harp/programme/course-information/nazareth/index.ejs

```diff
@@ -0,0 +1,10 @@
+<div class="section-copy">
```

Comment:
Okay I see we have the same code repeated twice, with the only small change. The idea of building templates is to not repeat ourselves like this? Can you find a way to refactor out this repetition? Happy to lend a hand if you need help with this!